Add-ons/UmbracoForms/Installation/ManualUpgrade-v8.md

```diff
@@ -8,9 +8,11 @@ needsV9Update: "true"
 # Manually upgrading forms
```

Comment:
No versionTo tag?

Reply:
We usually add the `versionTo` tag, when there's an article for the later versions, right?

<img width="269" alt="image" src="https://user-images.githubusercontent.com/24529086/171840045-27b42293-5b06-43ca-9ea5-e8e19af7b64b.png">